testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -43,3 +43,33 @@ def test_transform_StoR_fail():
 
     with pytest.raises(TypeError, match='S must be of type float32'):
         r = mda.lib.distances.transform_StoR(s, box)
+
+
+def test_capped_distance_noresults():
+    point1 = np.array([0.1, 0.1, 0.1], dtype=np.float32)
+    point2 = np.array([0.95, 0.1, 0.1], dtype=np.float32)
+
+    pairs, distances = mda.lib.distances.capped_distance(point1,
+                                                         point2,
+                                                         max_cutoff=0.2)
+
+    assert_equal(len(pairs), 0)
+
+
+def test_capped_distance_checkbrute():
+    box = np.array([1., 1., 1., 90., 90., 90.], dtype=np.float32)
```

Comment:
there's still more tests required here, eg checking array of points vs array of points, checking triclinic vs orthogonal boxes (can use a `pytest.parametrize` for this)

if we're using `np.random` we also need to seed the random generator so that the exact same test is ran each time.  (otherwise maybe there's a 1 in 1,000,00 chance that the test fails with certain random conditions)